mlflow/models/cli.py

```diff
@@ -188,6 +190,16 @@ def build_docker(model_uri, name, env_manager, mlflow_home, install_mlflow, enab
 
         docker run -p 5001:8080 -e DISABLE_NGINX=true "my-image-name"
 
+    If ``--model-uri`` is NOT specified:
+
+    Builds a generic Docker image that can serve an arbitrary MLflow model. When launching
```

Comment:
Should document it only supports pyfunc flavor model.

Reply:
Added